Documentation~/preferences.md

```diff
@@ -1,279 +1,109 @@
-
+Enable or disable ProBuilder experimental features:
 
+* **Tools** > **ProBuilder** > **Editors** > [**New Bezier Shape**](bezier.md)
+* **Tools** > **ProBuilder** > **Experimental** > **Boolean (CSG)**.
+* **Preferences** window > **ProBuilder** > **Store Mesh as Asset**.
 
+> **Warning:** Experimental features can have unpredictable results.
 
+| **Property** | **Function** |
+| :--- | :--- |
+| **Experimental Features Enabled** | Add the experimental features to the Editor. |
+| **Store Meshes as Asset** | Store meshes as standalone assets in the Project folder, rather than as part of the Scene asset. |
 
-<a name="general"></a>
+> **Note**: When you activate or deactivate experimental features, the Unity Editor has to recompile scripts, which causes a delay before changes apply.
 
 ## General
 
-Use these properties to set some basic options for ProBuilder.
-
-![General section](images/prefs_general.png)
+Set some basic options for ProBuilder.
 
-|**Property:** |**Description:** |
-|:---|:---|
-|__Show Action Notifications__ |Enable this option if you want ProBuilder to notify you when performing actions. |
-|<a name="autouvs"></a>__Auto Lightmap UVs__ |Enable this option to generate the UV2 channel after every geometry edit. This means you don't have to manually generate them every time the Mesh changes.<br/><br/>UV2 channel generation for Meshes is necessary for lighting, but can be time-consuming. If you are editing objects with large numbers of vertices, disable this to save resources. |
-|__Show Missing Lightmap UVs Warning__ |Enable this option to show a warning in the console if ProBuilder shapes are missing a valid UV2 channel when Unity performs a lightmap bake. |
-|__Show Handle Info__ |Enable this option to display the information for moving, rotating, and scaling deltas in the bottom right of the Scene view. <br /><br />**Note**: If you have the [Component Editor **Tools** panel](https://docs.unity3d.com/Manual/UsingCustomEditorTools.html#ToolModesAccessSceneViewPanel) open in the Scene view, it covers this information. Close the panel to display the information. |
-|<a name="info_overlay"></a>__Show Scene Info__ |Enable this option to display the Mesh information overlay in the top left of the Scene view. These details include overall face, vertex and triangle counts, and the number of elements currently selected:<br />![Scene information overlay](images/info_overlay.png) |
-|__Script Stripping__ |Enable this option to automatically remove the extra data ProBuilder stores in a Scene. This includes all ProBuilder scripts, so if you are using the runtime API you should disable this feature. |
-
-
-
-<a name="graphics"></a>
+| **Property** | **Function** |
+| :--- | :--- |
+| **Show Action Notifications** | Enable this option if you want ProBuilder to notify you when performing actions [[Jon: where, how, which actions?]]. |
```

Comment:
It used to just display in the Scene view as a notification, but I can't get it to work anymore. Checking if this is a bug.

Reply:
It is not a bug. It still works, but I think I broke it on my project.

Reply:
Okay, I can get it to give me an error when I try to cut: not enough elements selected for a cut. 

The thing is: it cuts. So it's showing me a lie.

Reply:
```suggestion
| **Show Action Notifications** | Show explanations for failed actions, like cutting a mesh. |
```

Reply:
Does it only show explanations for failed notifications? @modrimkus-unity, is this the case?

Reply:
Yes, what Jon said, it's not just for failures.

Reply:
Ah, interesting. I couldn't find an action it was pleased with.

Reply:
@modrimkus-unity what are rules for showing these notifications? Like, why do some actions get it and some don't?

Reply:
```suggestion
| **Show Action Notifications** | Enable this option to display a notification in the Scene view after you perform an action with ProBuilder. |
```